### PR TITLE
Auto restore

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@333fred

**Comment by tmat** [comment]
@dotnet/roslyn-analysis

File: nuget/Directory.Build.props [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
   <Import Project="..\Directory.Build.props"/>
-  <Import Project="$(RepoToolsetDir)Settings.props" />
+  <Import Project="$(RepoToolsetDir)Settings.props" Condition="Exists('$(RepoToolsetDir)Settings.props')" />
```
Comment (333fred) [inline nuget/Directory.Build.props]: And VS actually re-evaluates these when the repo toolset is restored?

Comment (tmat) [inline nuget/Directory.Build.props]: Yes

**Pull request merged**